src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -5,9 +5,24 @@
  * Do not remove no-field constructor
  */
 public class Truck extends Machine {
+    private int truckSize;
```

Comment:
Consider using access modifiers for the 'truckSize' field. Fields should be accessed through getters and setters and generally be kept private to follow the encapsulation principle.